pkgs/applications/misc/jquake/default.nix

```diff
@@ -13,11 +15,14 @@ stdenv.mkDerivation rec {
 
   sourceRoot = ".";
 
+  # JQuake emits a lot of debug-like messages on stdout. Either drop the output
+  # stream entirely or log them at 'user.debug' level.
+  logDestination = if logOutput then ''| logger -p user.debug''
+                   else ''> \/dev\/null'';
   postPatch = ''
-    # JQuake emits a lot of debug-like messages in console, but I
-    # don't think it's in our interest to void them by default. Log them at
-    # the appropriate level.
-    sed -i "/^java/ s/$/\ | logger -p user.debug/" JQuake.sh
+    # Redirect standard output to the selected destination by appending
+    # 'logDestination' to the JVM invocation line
+    sed -i "/^java/ s/$/ ${logDestination}/" JQuake.sh
```

Comment:
```suggestion
  # JQuake emits a lot of debug-like messages on stdout. Either drop the output
  # stream entirely or log them at 'user.debug' level.
  logDestination = 
    sed -i "/^java/ s/$/ ${if logOutput then "| logger -p user.debug" else "> \/dev\/null"}/" JQuake.sh
```

Reply:
I think I don't understand what `logDestination` is doing inside the script. Also, it felt like the command would have gotten messier if I were to inline the expansion of `logOutput`, but I guess not...

Reply:
> I think I don't understand what logDestination is doing inside the script.

My bad, we no longer need that.